### PR TITLE
Fix the bug that etcd txn proxy fails to print response revision crash

### PR DESCRIPTION
**Pull request by sky-big**
What type of PR is this?
Bug fixes

What this PR does / why we need it:
Fix kubebrain crash when etcd txn proxy fails to print response kv revision.

Detail: when the leader etcd txn create(or other action) fails, follower node print the revision of kv after receiving the error message from the leader, but the kv field is empty, so kubebrain process crash.

Which issue(s) this PR fixes:
None

Special notes for your reviewer:
None

### CODE REVIEW

File: pkg/server/service/etcdproxy/etcd_proxy.go [hunk 1]
```diff
@@ -209,12 +209,16 @@ func (e *etcdProxy) Txn(ctx context.Context, txn *etcdserverpb.TxnRequest) (*etc
 	}
 	if !resp.GetSucceeded() {
 		getResp := (*clientv3.GetResponse)(resp.Responses[0].GetResponseRange())
```
Comment (divanodestiny) [inline pkg/server/service/etcdproxy/etcd_proxy.go:211]: should check len of `resp.Responses` too？

Comment (sky-big) [reply pkg/server/service/etcdproxy/etcd_proxy.go]: Yes, it is. The leader's code responds to normal requests with responses, but in terms of defensive programming, it really needs to be judged

**Review by divanodestiny — APPROVED** [review]
LGTM

**Comment by divanodestiny** [comment]
@sky-big  could you sign the commit and push again ? thx~

**Comment by sky-big** [comment]
> @sky-big could you sign the commit and push again ? thx~

done.

**Comment by divanodestiny** [comment]
> > @sky-big could you sign the commit and push again ? thx~
> 
> done.

There may be something wrong, which block the merging. I think you can follow this doc to enable github to verify your sign.

**Comment by sky-big** [comment]
@divanodestiny  I have squashed this commit, see if you can merge this request, how to verify your sign?

**Comment by divanodestiny** [comment]
> @divanodestiny I have squashed this commit, see if you can merge this request, how to verify your sign?

sorry, i forget to write the url. it is [about-commit-signature-verification](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification).  

I can not merge this PR without breaking the branch protections. I think it would be better if you can follow the [guideline](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification) , config the public key on Github and enable Github to verify your sign for the commit. 

However, if you don’t want to do that, let me know that. Then i will try to force merge your PR. 

Thx for your work again ~

**Comment by sky-big** [comment]
> > @divanodestiny I have squashed this commit, see if you can merge this request, how to verify your sign?
> 
> @sky-big sorry, i forget to write the url. it is [about-commit-signature-verification](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification).
> 
> I can not merge this PR without breaking the branch protections. I think it would be better if you can follow the [guideline](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification) , config the public key on Github and enable Github to verify your sign for the commit.
> 
> However, if you don’t want to do that, let me know that. Then i will try to force merge your PR.
> 
> Thx for your work again ~

Done.

**Pull request merged**